tests/test_model_label.py

```diff
@@ -0,0 +1,52 @@
+"""Tests for model label construction checker."""
+import astroid
+from pylint.interfaces import HIGH
+from pylint.testutils import CheckerTestCase
+from pylint.testutils import MessageTest
+from pytest import mark
+
+from pylint_nautobot.model_label import NautobotModelLabelChecker
+
+from .utils import assert_no_message
+
+_COL_OFFSET = 11
+
+
+class TestModelLabelChecker(CheckerTestCase):
+    """Test model label construction checker."""
+
+    CHECKER_CLASS = NautobotModelLabelChecker
+
+    @mark.parametrize(
+        "test_string",
+        (
+            'f"{model._meta.app_label}.{model._meta.model}"',
+            'f"Prepended {another_model._meta.app_label}.{another_model._meta.model}, appended"',
+        ),
+    )
+    def test_finds_model_label_construction(self, test_string):
+        module_node = astroid.parse(f"def get_label(model, another_model):\n    return {test_string}")
+        fnode = module_node.body[0].body[0].value  # type: ignore
```

Comment:
Mind adding a comment explaining this line?

Reply:
Added